### PR TITLE
Removed pods from project

### PR DESCRIPTION
**Pull request by sprzenus**

### CODE REVIEW

File: Restler-Example/Restler-Example.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -4,7 +4,4 @@
    <FileRef
       location = "group:Restler-Example.xcodeproj">
    </FileRef>
-   <FileRef
-      location = "group:Pods/Pods.xcodeproj">
-   </FileRef>
 </Workspace>
```
Comment (PiotrPawlus) [inline Restler-Example/Restler-Example.xcworkspace/contents.xcworkspacedata]: Since you removed Cocoapods we don't need longer `xcworkspace`

**Comment by PiotrPawlus** [comment]
Please resolve the issue with running example app

**Comment by sprzenus** [comment]
> Please resolve the issue with running example app

Added configuration instructions in README and configuration script for a quicker and less painful project configuration.

File: .github/workflows/examle.yml [hunk 2]
```diff
@@ -30,21 +30,23 @@ jobs:
...
 
       - name: Clean
         run: xcodebuild clean | xcpretty
         working-directory: ${{ env.working-directory }}
         shell: bash
 
       - name: Build
-        run: xcodebuild build -scheme Restler-Example -workspace Restler-Example.xcworkspace | xcpretty --color
+        run: xcodebuild build -scheme Restler-Example -project Restler-Example.xcodeproj | xcpretty --color
         working-directory: ${{ env.working-directory }}
```
Comment (PiotrPawlus) [inline .github/workflows/examle.yml:51]: Should name of the file little bit different?

Comment (sprzenus) [reply .github/workflows/examle.yml:51]: of what file?

**Pull request merged**